index.bs

```diff
@@ -6004,9 +5985,9 @@ partial interface MLGraphBuilder {
     1. If |input|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dataType}} is not equal to |other|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dataType}}, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
     1. Let |descriptor| be a new {{MLOperandDescriptor}}.
     1. Set |descriptor|.{{MLOperandDescriptor/dataType}} to |input|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dataType}}.
-    1. Set |descriptor|.{{MLOperandDescriptor/dimensions}} to the result of running the [=MLGraphBuilder/broadcast-shapes=] steps given |input|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}} and |other|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}}.
-        1. If that [=exception/throws=] an error, re-[=exception/throw=] the error.
-    1. If |condition| is not unidirectionally broadcastable to |descriptor|.{{MLOperandDescriptor/dimensions}} according to the [[!numpy-broadcasting-rule]], then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
+    1. Set |descriptor|.{{MLOperandDescriptor/dimensions}} to the result of [=bidirectionally broadcasting the shapes=] |input|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}} and |other|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}}.
+        1. If that returns failure, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
+    1. If |condition| is not [=unidirectionally broadcastable=] to |descriptor|.{{MLOperandDescriptor/dimensions}}, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
```

Comment:
```suggestion
    1. If |condition| is not [=bidirectionally broadcastable=] to |descriptor|.{{MLOperandDescriptor/dimensions}}, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
```

Oops, existing spec typo, as the 3 input tensors (`condition`, `trueValues`, `falseValues`) are actually *multidirectionally* broadcastable with each other, which is functionally equivalent to `bidirectionalBroadcast(a, bidirectionalBroadcast(b, c))`. That's what the reference and implementation do:

https://github.com/webmachinelearning/webnn-baseline/pull/56/files#diff-1ab0e4e354fcd892d24b937bb8aad28636b554a25f248d3433573b57603ea411

```
  const valueShape = getBroadcastShape(trueValuesReshape.shape, falseValuesReshape.shape);
  const outputShape = getBroadcastShape(condition.shape, valueShape);
```

https://chromium-review.googlesource.com/c/chromium/src/+/5039655

```
  const auto value_shape =
      BroadcastShapes(true_value.dimensions, false_value.dimensions, true);
  if (!value_shape) {
    return base::unexpected(
        "The shapes of true_value and false_value are not broadcastable.");
  }

  const auto output_shape =
      BroadcastShapes(condition.dimensions, value_shape.value(), true);
  if (!output_shape) {
    return base::unexpected(
        "The condition shape is not broadcastable to the shape broadcasted "
        "from true_value and false_value.");
  }
```

https://chromium-review.googlesource.com/c/chromium/src/+/5069382

```
// * The shapes of all input operands must be bidirectionally broadcastable. (with each other)
```